### ISSUE TITLE
oclint-json-compilation-database takes very long time

### ISSUE THREAD

**Issue by buptwsg** [oclint/oclint#539]
### Issue Summary
oclint-json-compilation-database -v -e Pods -- -report-type html -o oclintReport.html takes 
very long time to finish, but it finished very soon a few days ago.

### Environment
- Operation system name and version: Mac OS 10.14.4
- OCLint version: 0.13.1
- How OCLint is installed: local build? prebuilt binary downloaded from github? homebrew install? others?
  local build

### Reproduction Steps
Detailed steps to reproduce the issue.

#### Sample code

```
C
C++
Objective-C
```

Command to run OCLint with the code above:

xcodebuild clean -scheme iOSTests -workspace iOSTests.xcworkspace -sdk iphonesimulator && xcodebuild build COMPILER_INDEX_STORE_ENABLE=NO -scheme iOSTests -workspace iOSTests.xcworkspace -sdk iphonesimulator -dry-run -UseNewBuildSystem=NO | xcpretty -r json-compilation-database --output compile_commands.json

oclint-json-compilation-database -v -e Pods -- -report-type html -o oclintReport.html

### Expected Behavior
oclint-json-compilation-database should finish in a short time for a simple test iOS project.

### Actual Behavior
The command takes a long time to finish, I noticed that oclintReport.html is generated very quickly, but the command doesn't finish after the report is generated.

It's also very strange that, if the computer is not connected to network, then everything is fine.

**Comment by dzoanb** [comment]
We had the same issue. iOS app in Objective-C, OCLint 0.13.1

**Fix:** Add a `-no-analytics` param somewhere after the `--` so it is passed to the `oclint` command. 

**Synopsis:**
An iOS project in Obj-C that took around 20 minutes to lint jumped to 1,5h or more overnight on 3rd of May. We excluded the possibility of our scripts causing the issue and environment changes. The CPU usage was the thing that got us thinking since after a few minutes of intense 98% it was dropping to 0% and hung like that for up to 10 minutes. One of my colleagues joked that maybe OCLint's analytics are down so when I started going through the source code as the last resort and saw a check for analytics I just had to give it a try. It turned out disabling the analytics indeed fixed the linting time.
